test/PooltogetherGovernor.t.sol

```diff
@@ -1124,3 +1131,297 @@ contract CastVoteWithReasonAndParams is ProposalTest {
     }
   }
 }
+
+contract _Execute is ProposalTest {
+  function setUp() public virtual override(ProposalTest) {
+    ProposalTest.setUp();
+    _upgradeToBravoGovernor();
+  }
+
+  function _randomUniswapPosition(uint256 _seed) internal returns (uint256 _token) {
+    if (_seed % 6 == 0) _token = 454861;
+    if (_seed % 6 == 1) _token = 454865;
+    if (_seed % 6 == 2) _token = 326807;
+    if (_seed % 6 == 3) _token = 326798;
+    if (_seed % 6 == 4) _token = 326783;
+    if (_seed % 6 == 5) _token = 321228;
+  }
+
+  function testFuzz_UpdateV4DripRate(uint256 newDrip) public {
+    // Drip must be greater than 0
+    // https://etherscan.io/address/0xbd537257fad96e977b9e545be583bbf7028f30b9#code#F1#L162
+    vm.assume(newDrip > 0);
+
+    IV4PooltogetherTokenFaucet tokenFaucet = IV4PooltogetherTokenFaucet(V4_TOKEN_FAUCET);
+    uint256 oldDrip = tokenFaucet.dripRatePerSecond();
+    assertEq(oldDrip, 5_787_037_037_037_037, "Old value is not correct");
```

Comment:
Why do we need this assertion?

Reply:
hmmm, yes I guess we could remove as it is a fuzz test.

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/15/commits/6e5b21a9923b1422141be80e914489f6a954e4db